terraform/outputs.tf

```diff
@@ -0,0 +1,66 @@
+# CC006 mandatory outputs
+output "app_name" {
+  description = "Name of the main Kafka application"
+  value = var.deployment_mode == "single" ? (
+    length(juju_application.kafka_single) > 0 ? juju_application.kafka_single[0].name : null
+  ) : (
+    length(juju_application.kafka_broker) > 0 ? juju_application.kafka_broker[0].name : null
+  )
+}
+
+output "endpoints" {
+  description = "Service access endpoints"
+  value = {
+    # Add actual service URLs here if available
+    # For now, this could include bootstrap servers, etc.
+  }
+}
+
+output "provides_endpoints" {
+  description = "Relation endpoints this charm provides"
+  value = [
+    "kafka-client",
+    "cos-agent",
+    "certificates"
```

Comment:
`certificates` is not a requires endpoint. I would either remove it, or create a new output `requires_endpoints`. thb, I can't think why would be useful to have a `requires` key, so maybe we can remove it.
Also, check this for the other modules.